StyleLintを追加する

shikky0331 opened the pull request:
close #13 

https://stylelint.io/user-guide/get-started を参考にStyleLintを導入しました。

Commit:
feat: add stylelint and stylelint-config-standard

Commit:
feat: add stylelint-config-sass-guidelines

Commit:
feat: add stylelint-order

Commit:
feat: add stylelint --fix command to lint-staged and lint scripts

piro0919 commented on .stylelintrc.json:
```diff
@@ -0,0 +1,19 @@
+      "custom-properties",
+      "declarations"
+    ],
+    "order/properties-order": [
+      "width",
+      "height"
+    ]
```
[imo]

あくまで個人的にはですが `properties-alphabetical-order` のほうがシンプルでわかりやすいかなーと思いますが、どうでしょうか？ 🦊 
遠藤さんの好みのおまかせしますがー。

rel https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-alphabetical-order/README.md

shikky0331 replied:
@piro0919 
確かに`properties-alphabetical-order`のアルファベット順の方が理解しやすいですね！

こちらで変更しました🙇‍♂️
https://github.com/shikky0331/shikky/pull/16/commits/8164f2799df24605c6326d2db5854aa5103f9e84

piro0919 commented on package.json:
```diff
@@ -28,10 +28,11 @@
     "prepare": "husky install",
     "commit": "cz",
-    "lint": "eslint src/**/*.{ts,tsx} --fix"
+    "lint": "eslint src/**/*.{ts,tsx} --fix; stylelint src/**/*.scss --fix"
```
[must]

この書き方では `eslint` もしくは `stylelint` のみを実行することができないので良くないですね。 🙅‍♂️ 
例えば `lint:style` など、新しいスクリプトとして登録するようにしましょう。 👍

shikky0331 replied:
@piro0919 
こちらで修正いたしました🙇‍♂️
https://github.com/shikky0331/shikky/pull/16/commits/632ba37cbe594dad1897b454742890c763838642

Commit:
chore: run commands separately for eslint and stylelint

Commit:
chore: change order/properties-order to order/properties-alphabetical-order

Because properties-alphabetical-order is easier to understand than order/properties-order

piro0919 reviewed (approved):
LGTM 👍👍👍
